src/components/Images/ImageBase/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { BaseSyntheticEvent, CSSProperties } from "react";
+
+import BoxBase, { BoxBaseProps } from "src/components/Boxs/BoxBase";
+
+export interface ImageBaseProps extends BoxBaseProps {
+    onErrorSrc: string;
+    src: string;
+    alt?: string;
+    styleImage?: CSSProperties;
+}
+
+const ImageBase = (props: ImageBaseProps) => {
+    const {
+        onErrorSrc,
+        styleImage,
+        src,
+        alt,
+        sizeWidth = null,
+        sizeHeight = null,
```

Comment:
I think we should set a default size here

Reply:
Or you can force these props in the interface

Reply:
![image](https://user-images.githubusercontent.com/58205909/131253133-281a5026-1fd5-4e8e-92ea-ca1f1efa4258.png)
I will ignore they in the interface